README.md

```diff
@@ -3,23 +3,24 @@ The Container Storage Interface (CSI) Driver for IBM block storage systems enabl
 
 This is the official operator to deploy and manage IBM block storage CSI driver.
 
-Supported container platforms:
-  - OpenShift v4.4
-  - OpenShift v4.5
-  - Kubernetes v1.17
-  - Kubernetes v1.18
+Supported container platforms (and architectures):
+  - OpenShift v4.3 (IBM Z and IBM PowerPC)
+  - OpenShift v4.4 (x86 and IBM Z)
+  - OpenShift v4.5 (x86)
+  - Kubernetes v1.17 (x86)
+  - Kubernetes v1.18 (x86)
 
 Supported IBM storage systems:
   - IBM Spectrum Virtualize Family (including IBM Flash family members built with IBM Spectrum Virtualize (FlashSystem 5010, 5030, 5100, 7200, 9100, 9200, 9200R) and IBM SAN Volume Controller (SVC) models SV2, SA2)
```

Comment:
update to:
IBM Spectrum Virtualize Family including IBM SAN Volume Controller (SVC) and IBM FlashSystem® family members built with IBM Spectrum® Virtualize (FlashSystem 5010, 5030, 5100, 7200, 9100, 9200, 9200R)

Reply:
done